src/jit/valuenum.cpp

```diff
@@ -5858,8 +5916,9 @@ void Compiler::fgValueNumberTree(GenTreePtr tree, bool evalAsgLhsInd)
                                     // Now get rid of any remaining struct field dereferences. (if they exist)
                                     if (fldSeq->m_next)
                                     {
-                                        storeVal = vnStore->VNApplySelectorsAssign(VNK_Liberal, fldMapVN,
-                                                                                   fldSeq->m_next, storeVal, indType, compCurBB);
+                                        storeVal =
+                                            vnStore->VNApplySelectorsAssign(VNK_Liberal, fldMapVN, fldSeq->m_next,
+                                                                            storeVal, indType, compCurBB);
```

Comment:
Yes, I think there were some commits that didn't have jit-format run on them; sorry about that.

---
In reply to: [77282298](https://github.com/dotnet/coreclr/pull/7027#discussion_r77282298) [](ancestors = 77282298)